allennlp/models/vision_text_model.py

```diff
@@ -231,7 +287,10 @@ def forward(
         else:
             raise ValueError(f"Fusion method '{self.fusion_method}' not supported")
 
+        # Shape: (batch_size, num_labels)
         logits = self.classifier(pooled_output)
+
+        # Shape: (batch_size, num_labels)
```

Comment:
We should find some time to experiment with named tensor dimensions. I think a lot of our code could be much more readable if we used them.